new-file.md

```diff
@@ -0,0 +1 @@
+This is a new file. * It is formatted using markdown. * That's why it has the `.md` file extension.
```

Comment:
Actually I am just trying to make in-line comment for my own file to understand the PR steps better